test/custom-formatter-test.cc

```diff
@@ -5,6 +5,12 @@
 //
 // For the license information refer to format.h.
 
+#ifdef _MSC_VER
```

Comment:
Here and below: I'd drop `_MSC_VER_` altogether. There is no harm in always defining `_CRT_SECURE_NO_WARNINGS`.

Reply:
Done.

Anyway I notice `format-imple-test` is exempted when building DLL. When building DLL it includes `format-inl.h` in which cout_digits resides, I wonder if `format-imple-test` should be re-enabled also?